### ISSUE TITLE
OpenYeast plasmids appearing in synthesis FASTA

### ISSUE THREAD

**Issue by jakebeal** [iGEM-Engineering/iGEM-distribution#199]
Because we don't currently have a good way to handling the import of the OpenYeast plasmids, they are appearing in the FASTA for synthesis. As such, any synthesis run needs to exclude them.

**Comment by vinoo-igem** [comment]
Yup, my plan is to manually omit these. #193 may also be helpful in the future in this aspect for "already synthesized/sample exists."
